sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3417,8 +3417,7 @@ object GpuOverrides extends Logging {
           val aggBuffer = c.aggBufferAttributes.head
           aggBuffer.copy(dataType = CudfTDigest.dataType)(aggBuffer.exprId, aggBuffer.qualifier)
         }
-      }).incompat("the GPU implementation of approx_percentile is not bit-for-bit " +
```

Comment:
But we are not 100% compatible with the Spark CPU version. We have to document this some where. Even if we as a group are okay with prioritizing performance over bit for bit identical results, we still need to document that this is different some how.

Reply:
@revans2 Could you take another look? Please see the updated PR description.